package.json

```diff
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "chai": "~1.5.0",
+    "debug": "~3.2.6",
```

Comment:
Was this a missing dependency or perhaps when testing locally? I can't seem to find where it's used

Reply:
Yes, sorry, forgot to mention this. This dev dependency is used by `mocha`. Automatically `mocha` picks the latest `debug` lib verison 4.x.x which uses features from ES2015 (like `let` keywords) and this is not compatible with node v4 (supported by this library). By pinpointing the `debug` version to 3.x.x. (3.2.6 is the latest) we can still successfully run tests for nodejs v4.